tools/functional-tester/etcd-runner/command/election_command.go

```diff
@@ -15,11 +15,10 @@
 package command
 
 import (
-	"context"
 	"errors"
-	"fmt"
+	"log"
 
-	"github.com/coreos/etcd/clientv3/concurrency"
+	rl "github.com/coreos/etcd/tools/functional-tester/etcd-runner/lib"
```

Comment:
Don't call packages "lib". It's not descriptive at all and not idiomatic go. Try `runner`.